<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>430</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_system">
     <attribute name="title">
      <string>System</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>System Information</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>System</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kernel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Boot Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Running Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_system">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_kernel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_boot">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_time">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU Model</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU MHz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Memory</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> MemFree</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> MemAvailable</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>SwapCached</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_memtotal">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_memfree">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_memavailable">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_swapcached">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>350</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cpumodel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Intel(R) Core(TM) i5-4210M CPU @ 2.60GHz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_cpumhz">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_shutdown">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>340</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>shutdown</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_charts">
     <attribute name="title">
      <string>Charts</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>401</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>CPU</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>0</y>
          <width>601</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_cpu"/>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Swap</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>-1</y>
          <width>601</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_swap"/>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Mem</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>-1</y>
          <width>601</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_mem"/>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Process</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-10</y>
        <width>611</width>
        <height>411</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Process List</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_process">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>600</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Process Kill</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_killProcess">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>130</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Search">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>160</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Kill">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>160</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Kill</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>130</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Pid/Name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_search_process_info">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>210</y>
          <width>281</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>System Time :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SysTime">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>200</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>85</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU Usage :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cpuusage">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>430</y>
      <width>85</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>54.67%</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
